Remotion/Validation/Core/Attributes/Validation/LengthValidationAttribute.cs

```diff
@@ -59,7 +59,7 @@ protected override IEnumerable<IPropertyValidator> GetValidators (IPropertyInfor
       ArgumentUtility.CheckNotNull ("property", property);
       ArgumentUtility.CheckNotNull ("validationMessageFactory", validationMessageFactory);
 
-      LengthValidator validator;
+      LengthValidator? validator;
```

Comment:
Because it is just a declaration with no value assigned. At this point the value of the local variable is `null`. Interestingly enough, there is no compiler error emitted when omitting the `?`. I don't know when this was changed or if this is a bug, so I'm not sure if this should be changed, I'll research this.